manim/mobject/types/vectorized_mobject.py

```diff
@@ -708,6 +712,7 @@ def set_shade_in_3d(
                 submob.z_index_group = self
         return self
 
+    @internal
```

Comment:
boderline?

Reply:
I mostly added this because of discussions on whether to make `points` a property (to add some memoization stuff).